mxd/backend-service/src/main/java/org/eclipse/mxd/Main.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+package org.eclipse.mxd;
+
+import org.eclipse.mxd.service.ContentService;
+import org.eclipse.mxd.service.Impl.ContentServiceImpl;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.eclipse.mxd.util.HibernateUtil;
+import org.glassfish.jersey.servlet.ServletContainer;
+import org.hibernate.cfg.Configuration;
+
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class Main {
+    private final static ContentService contentService = new ContentServiceImpl();
+    private static final Logger logger = Logger.getLogger(Main.class.getName());
+
+    public static void main(String[] args) {
+        startJettyServer();
+    }
+
+    private static void startJettyServer() {
+        Properties appProperties = new Properties();
```

Comment:
Create a `SettingResolver` class with `initialize()` and `getSetting(String key)` methods.
Inside initialize() load all properties into a `Map<String, String>` then load all `System.getEnv()` into the same map. (envs will have preference, so it will override properties).

Reply:
SettingResolver  is created with initialize() and getSetting(String key) methods is completed.